docs/plugins/feature-flags.md

```diff
@@ -16,29 +16,30 @@ Defining feature flag in a plugin is done by passing the name of the feature fla
 
 ```ts
 /* src/plugin.ts */
-import { createPlugin, createRouteRef } from '@backstage/core-plugin-api';
-import ExampleComponent from './components/ExampleComponent';
+import { createPlugin } from '@backstage/core-plugin-api';
 
 export const examplePlugin = createPlugin({
-  id: 'example',
-  routes: {
-    root: rootRouteRef,
-  },
+  // ...
   featureFlags: [{ name: 'show-example-feature' }],
+  // ...
 });
 ```
 
 ### In the application
 
-Defining feature flag in the application is done by adding feature flags in`featureFlags` array in
+Defining feature flag in the application is done by adding feature flags in `featureFlags` array in
 `createApp()` function call:
 
 ```ts
+import { createApp } from '@backstage/app-defaults';
```

Comment:
The reason I added the import was because I was confused as to what the createApp function was referencing and there was inconsistency in the code examples on when imports were being referenced. It seems to me that perhaps it would always be best when importing something from backstage to include that import in the code example. Granted, I'm just getting fresh back into Backstage after like 5 years so I'm pretty green :D

Reply:
Ah, that makes sense based on this additional context. Let's keep the import and add the title - win, win!